docs/pywbemcli/cmdshelp.rst

```diff
@@ -2302,17 +2301,18 @@ Help text for ``pywbemcli statistics server-off`` (see :ref:`statistics server-o
 
     Usage: pywbemcli [GENERAL-OPTIONS] statistics server-off [COMMAND-OPTIONS]
 
-      Disable statistics on current server.
+      Disable server maintained statistics.
 
-      This command turns off the gathering of statistics in the WBEM server and the return of server response times for
-      inclusion in pywbemcli maintained statistics. The gathering of server statistics and the returning of server
-      response times may not be implemented by all WBEM servers.
+      This command deactivates the gathering of statistics in the current WBEM server and also deactivates the returning
+      of server response times in the CIM-XML response for inclusion in the client maintained statistics, by setting the
+      'GatherStatisticalData' property to False in the 'CIM_ObjectManager' instance for the WBEM server.
 
-      This may fail if the server does not manage statistics or does not allow a client to modify the state of statistics
-      gathering.
+      This command will fail if the server does not support gathering statistics or does not allow a client to modify the
```

Comment:
Can we change "command will fail' to 'command fails'

Reply:
DONE